src/main/java/lotto/LottoGame.java

```diff
@@ -4,22 +4,34 @@
 import lotto.view.InputView;
 import lotto.view.ResultView;
 
+import java.util.ArrayList;
 import java.util.List;
 
-public class LottoGame {
+import static lotto.domain.LottoMachine.PRICE_OF_LOTTO;
 
+public class LottoGame {
     public static void main(String[] args) {
         long amount = InputView.purchaseAmount();
+
+        int manualPurchaseLottoCount = InputView.inputManualPurchaseCount();
+        if (manualPurchaseLottoCount * PRICE_OF_LOTTO > amount) {
+            throw new IllegalArgumentException("수동 구매 가격이 구입금액보다 클 수 없습니다.");
+        }
+
+        List<String> manualLottoNumberInputs = new ArrayList<>();
+        if (manualPurchaseLottoCount > 0) {
+            manualLottoNumberInputs = InputView.inputManualLottoNumbers(manualPurchaseLottoCount);
+        }
```

Comment:
InputView에서 빈 리스트를 반환하도록 수정하면 코드가 더 간결해질 것 같아요 🤔

```suggestion
        List<String> manualLottoNumberInputs = InputView.inputManualLottoNumbers(manualPurchaseLottoCount);
```

이렇게 하면 `inputManualLottoNumbers` 메서드 내부에서 `manualPurchaseLottoCount`가 0 이하일 경우 빈 리스트를 반환하도록 구현하면 될 것 같아요! 
입력 처리 로직이 `InputView`에 완전히 캡슐화되어 책임 분리가 더 명확해질 것 같습니다! 👍